Makefile

```diff
@@ -9,6 +9,11 @@ export PATH := $(shell pwd)/tmp:$(PATH)
 install:
 	 mkdir -p tmp;(cd tmp; git clone --depth=1 https://github.com/fredrikhgrelland/vagrant-hashistack.git; cd vagrant-hashistack; make install); rm -rf tmp/vagrant-hashistack
 
+check_for_consul_binary:
+ifeq (, $(shell which consule))
```

Comment:
```suggestion
ifeq (, $(shell which consul))
```

Reply:
😱 Forgot to remove after testing. Thanks for finding that!!